### PR TITLE
LPS-74800

### PR DESCRIPTION
**Pull request by jcampoy**
Created basic AssetRenderers for Sites and Organizations based on the UserAssetRenderer. cc/ @JorgeFerrer @ricardocousolr

### CODE REVIEW

File: modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/servlet/taglib/ui/OrganizationCategorizationFormNavigatorEntry.java [hunk 1]
```diff
@@ -43,15 +43,6 @@ public String getKey() {
 	}
 
 	@Override
```
Comment (jcampoy) [inline modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/servlet/taglib/ui/OrganizationCategorizationFormNavigatorEntry.java:43]: We need to specify the categories when you are creating the organizations if you have a required vocabulary for a Organization, so, we need to display also the categorization when we are creating it.

**Comment by pavel-savinov** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/51680. [See changes here.](https://github.com/pavel-savinov/liferay-portal/compare/4251e44~2...4251e44) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pavel-savinov** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/51685. [See changes here.](https://github.com/pavel-savinov/liferay-portal/compare/ece1c03~3...ece1c03) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**